modules/dkan/dkan_datastore/includes/DkanDatastore.inc

```diff
@@ -341,18 +280,49 @@ class DkanDatastore extends Datastore implements DatastoreFormInterface {
    * Drop form submision.
    */
   public function dropFormSubmit(&$form_state) {
+    $this->dropDataStore();
+    drupal_set_message(t('Datastore dropped!'));
```

Comment:
Can we include the resource name in the message?

Reply:
OK, updated the title... btw above was just a refactor so "Datastore dropped!" was the original message for drops.